apps/platform-app/src/app/[locale]/(profile)/(components)/LocaleSwitcher/LocaleSwitcherSelect.tsx

```diff
@@ -0,0 +1,129 @@
+'use client'
+
+import { useMediaQuery } from '@/hooks/useMediaQuery'
+import useOutsideClick from '@/hooks/useOutsideClick'
```

Comment:
**suggestion**: Ensure that the useOutsideClick hook is necessary and functioning as expected. If not, consider removing it.